sw/host/opentitanlib/src/util/bigint.rs

```diff
@@ -113,15 +113,25 @@ impl<const BIT_LEN: usize, const EXACT_LEN: bool> fmt::Display
     }
 }
 
+impl<const BIT_LEN: usize, const EXACT_LEN: bool> From<FixedSizeBigInt<BIT_LEN, EXACT_LEN>>
+    for String
+{
+    fn from(bigint: FixedSizeBigInt<BIT_LEN, EXACT_LEN>) -> String {
+        bigint.to_string()
+    }
+}
+
```

Comment:
I thought implementing `fmt::Display` would automatically provide `to_string()` and we could use that if needed. I assume this is required for Serde serialization, right? Can you look into if we can somehow use `to_string()` instead of implementing this here and in the macro below?